### DISCUSSION TITLE
Please get the Firefox's AMO Recommended badge

### DISCUSSION THREAD

**Discussion by OutshineIssue** [filips123/PWAsForFirefox#497] [Ideas]
I've had my doubts about using this addon for a while now, and with the recent pre-[2.12.0](https://github.com/filips123/PWAsForFirefox/releases/tag/v2.12.0) vulnerability, those doubts have only escalated. I strongly suggest that you do what it takes to get the [Recommended badge](https://support.mozilla.org/en-US/kb/recommended-extensions-program) from Firefox, because as described by Mozilla, "Extensions undergo intense scrutiny, including manual security reviews".

> This add-on is not actively monitored for security by Mozilla. Make sure you trust it before installing.

This is currently a warning on the PWAsForFirefox AMO page. You can imagine how this simple paragraph can scare people away from installing an addon with such critical permissions.

**Comment by filips123** [comment]
Well, are recommended extensions reviewed only for intentional malicious activities or also for unintentional bugs and security issues? Also, the recent vulnerability was in the native program, so do they actually review it and all other related scripts, or only the actual extension?

One problem with recommended extensions is that updates are often delayed because of reviews. So, if the user already updates the native program to the latest release while the extension is still being reviewed, they will get a warning that the extension is outdated, and might experience issues in case of breaking changes (that aren't often, but still). Also, if there is an important fix (for example, for compatibility with the latest Firefox version), it might get delayed.
